ArduPlane/Attitude.cpp

```diff
@@ -58,7 +58,9 @@ bool Plane::stick_mixing_enabled(void)
         // we're in an auto mode. Check the stick mixing flag
         if (g.stick_mixing != STICK_MIXING_DISABLED &&
             g.stick_mixing != STICK_MIXING_VTOL_YAW &&
-            geofence_stickmixing() &&
+            #if AC_FENCE == ENABLED
```

Comment:
Probably not a blocker but we often put the #if left justified..

Reply:
Ack, also removed `#if` in `if` statement.